source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,16 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.15.0:
+
+{+atlas-cli+} 1.15.0
+--------------------
+
+Released 22 February 2024
+
```

Comment:
can you please add 2 more items:

- Adds support for Atlas search nodes management: atlas cluster search nodes [list/create/update/delete]
- Add --tag flag to "atlas projects create" command